### PR TITLE
Revert "Say "Calculated Event" instead of "Action""

### PR DESCRIPTION
**Pull request by yakkomajuri**
This reverts commit b573888af57bac5222c62dba8e91fd3f053f49d1.

### CODE REVIEW

**Comment by Twixes** [comment]
Thx for catching this!

**Comment by Twixes** [comment]
Published integration version 2.0.2 to release this to users.

File: src/triggers/action_performed.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
     ORGANIZATION_AND_PROJECTS_FIELDS,
 } from '../utils'
 
-function getActionPerformance(_: ZObject, bundle: Bundle) {
+function getActionPerformance(z: ZObject, bundle: Bundle) {
```
Comment (lharries) [inline src/triggers/action_performed.ts:10]: this part seems out of scope?
